imgui/imgui.h

```diff
@@ -2312,21 +2282,27 @@ struct ImDrawListSplitter
     IMGUI_API void              SetCurrentChannel(ImDrawList* draw_list, int channel_idx);
 };
 
-enum ImDrawCornerFlags_
+// Flags for ImDrawList functions
+// (Legacy: bit 0 must always correspond to ImDrawFlags_Closed to be backward compatible with old API using a bool. Bits 1..3 must be unused)
+enum ImDrawFlags_
```

Comment:
This is an incompatible change not reflected in the wrapper.
Please figure out a way to make a compatible extension in the wrapper - I'd like to avoid bumping major versions for any minute change in the library.
Please cross-check also all affected function signatures.

Reply:
Good catch. I guess all functions receiving the flag needs to convert from the old to the new.
Should we also add new functions that accept the new enum?

Reply:
If it's possible to keep the same functions, yet with the new type, that would be great.

It should be by having the new flag type with the old constants being of that new type, in parallel to the new constants. The functions have the new flag type in their signature. Unless someone did some explicit typing in their code, this should work without a compilation issue.

In sum I'm OK with some compatibility redirects that work "automatically". I want to avoid creating an entire wrapper for compatibility in what is already a wrapper. 

So, something like:
```
type NewFlags int
type OldFlags int

const (
  NewFlagsConstant1 = NewFlags 1
  NewFlagsConstant2 = NewFlags 2
  ...
  // OldFlagsConstant1 does ...
  // Deprecated: Use NewFlagsConstant1. Will be removed in v5.
  OldFlagsConstant1 = NewFlagsConstant1 -- if applicable
)

func SomeFunc(..., flags NewFlags, ...) -- changing the signature from OldFlags
```

It's tricky to keep up with version compatibility and meaningful semantic versioning if the underlying library changes so often :/

Reply:
Did a attempt at this, it was easier than expected because imgui also tries to maintain backwards compatibility, so I could just copy the upstream definitions